### DISCUSSION TITLE
Welcome to adaptive_dialog Discussions!

### DISCUSSION THREAD

**Discussion by mono0926** [mono0926/adaptive_dialog#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mono0926** [comment]
test

**Reply by mono0926** [reply]
test

**Comment by Jackyto** [comment]
Hi, I dont know where to ask that but I'll give it a shot here
Is there a way to start an async request on showTextInputDialog validation ? I do not see any ways to implement it and I thought I could find help here
Thanks in advance
Cheers
Pierre

**Reply by mono0926** [reply]
That is currently not possible.
For now, please implement it concurrently without this package.

**Reply by Jackyto** [reply]
Okay, thank you for your answer, I did not expected so fast haha
Your library seems very nice, can't wait to explore it
Cheers mate
